ansible/setup.yml

```diff
@@ -105,9 +105,15 @@
         - ./configure
         - make
         - make install
-        - ldconfig
       args:
         chdir: "/usr/local/src/libsodium-{{ libsodium_version }}"
         creates: "/usr/local/lib/libsodium.so.{{ libsodium_module_version }}"
       tags:
         - libsodium
+      register: installed
+
+    - name: Run ldconfig
```

Comment:
Is this independent from AWS? Could it be merged independently?
I am not suggesting to put in more work, but if independent please mention that in commit message when merging PR to master cc @dincho

Reply:
without this, AWS deployment is failing

Reply:
@lucafavatella It's old forgotten bug that needs to be fixed anyway. So let's keep it here